processes/ingest/hathi_trust.py

```diff
@@ -36,44 +37,50 @@ def runProcess(self):
 
         logger.info(f'Ingested {len(self.records)} Hathi Trust records')
 
-    def importRemoteRecords(self, fullOrPartial=False):
-        self.importFromHathiTrustDataFile(fullDump=fullOrPartial)
+    def importRemoteRecords(self, start_date_time=None, full_or_partial=False):
+        self.importFromHathiTrustDataFile(start_date_time, full_dump=full_or_partial)
```

Comment:
Also this is just a pass through to another function in this class. Can we just call importFromHathiTrustDataFile?

Reply:
I agree that the method here is unnecessary so I'll remove it and call `importFromHathiTrustDataFile` directly in the `runProcess` method.